oneapp-server/dao/LocationDao.js

```diff
@@ -47,6 +47,11 @@ class LocationDao extends SQLDataSource {
       COUNTY_NUMBER: result ? result.COUNTY_NUMBER : null,
     };
   }
+
+  static isValidAddress(input) {
+    // TODO: fill in functionality
+    return input !== null;
+  }
```

Comment:
This is the method call that's going to the USPS address validation, correct? If so, I don't think it makes sense to have this in the DAO, since it won't need to interact with the database.

Could we create a `USPSValidationService` with a method signature of `isValidAddress(address, zip)`? This would match the SOAP API available to NJ already. The return value would be the same object fields as `AddressInput` or it would throw a `OneAppError` using transition text id `t2313` (this should match invalid address).

Since we're skipping actually integrating with SOAP API for the MVP, this method could contain just an object map array of a some different results, like so, and if a result isn't found in the array, return the OneApp error.

```
[
  {address: 'x', zip: '00000', result: { address1: '', address2: '', ...}}
]
```